### PR TITLE
Update MainActivity.java

### PR DESCRIPTION
**Pull request by TenzinJams**
i have added a commet

### CODE REVIEW

**Review by TenzinJams — COMMENTED** [review]
Let's try something new

**Review by Tandin02200166 — COMMENTED** [review]
Well Done

**Comment by TenzinJams** [comment]
oii mashey sa nga dhi gotuh nawmay

File: MainActivity.java [hunk 1]
```diff
@@ -12,4 +12,5 @@ protected void onCreate(Bundle savedInstanceState) {
     }
     //yaya
     //tub tub
+    //mishey bay may
```
Comment (TenzinJams) [inline MainActivity.java:15]: we HAVE TO ADD A METHOD HERE Wai.

**Pull request merged**